src/test/kotlin/day05/WandTest.kt

```
@@ -0,0 +1,95 @@
+import org.junit.Before
+
+import org.junit.Assert.*
+import org.junit.Test
+
+import org.example.day05.Wand
+
+class WandTest {
+
+    @Before
+    fun setUp() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)